README.md

```diff
@@ -1,23 +1,27 @@
 # protobuf-elixir
 
-[![Hex.pm](https://img.shields.io/hexpm/v/protobuf.svg)](https://hex.pm/packages/protobuf)
 [![Build Status](https://travis-ci.org/tony612/protobuf-elixir.svg?branch=master)](https://travis-ci.org/tony612/protobuf-elixir)
+[![Module Version](https://img.shields.io/hexpm/v/protobuf.svg)](https://hex.pm/packages/protobuf)
+[![Hex Docs](https://img.shields.io/badge/hex-docs-lightgreen.svg)](https://hexdocs.pm/protobuf/)
+[![Total Download](https://img.shields.io/hexpm/dt/protobuf.svg)](https://hex.pm/packages/protobuf)
+[![License](https://img.shields.io/hexpm/l/protobuf.svg)](https://github.com/tony612/protobuf-elixir/blob/master/LICENSE)
+[![Last Updated](https://img.shields.io/github/last-commit/tony612/protobuf-elixir.svg)](https://github.com/tony612/protobuf-elixir/commits/master)
```

Comment:
Please revert all of these changes, they are unnecessary in the README. Thanks!

Reply:
Noted. I will just put the GitHub Action badge.